test/Concurrency/global_actor_function_types.swift

```diff
@@ -130,12 +130,14 @@ func testTypesConcurrencyContext() async {
   let _: () -> Int = f2 // expected-error{{converting function value of type '@SomeGlobalActor () -> Int' to '() -> Int' loses global actor 'SomeGlobalActor'}}
 
   // expected-error@+1{{expression is 'async' but is not marked with 'await'}}{{7-7=await }}
-  _ = f1() //expected-note{{call is 'async}}
+  _ = f1() //expected-note{{calls to let 'f1' from outside of its actor context are implicitly asynchronous}}
   // expected-error@+1{{expression is 'async' but is not marked with 'await'}}{{7-7=await }}
-  _ = f2() //expected-note{{call is 'async'}}
+  _ = f2() //expected-note{{calls to let 'f2' from outside of its actor context are implicitly asynchronous}}
```

Comment:
tiny nit: `calls to let ‘f2’` reads a little odd, could it be something like “calls of closure ‘f2’…”?

Reply:
I agree that `let` isn't great. I'm wondering if `closure` might be unexpected though. Do people think of unapplied function references as closures?

Reply:
That's a good point, hmm...I wonder if changing `let` to `constant` and `var` to `variable` would make this read nicer. `calls of constant 'f2'...`

Reply:
But if there's no prior art in the docs for `constant` to refer to `let`s, then maybe it's not worth introducing that...

Reply:
Maybe something like `closure reference` or a `function reference` depending on what we're referencing?

e.g `calls to function reference 'f1' from outside...`

Reply:
I'm going to keep thinking about this. There are other diagnostics (even near this line in the same file, basically anything that uses `DescriptiveDeclKind`) that use `let` and it's strange to read, so it's worth updating all of them together.